ReactWindows/ReactNative/ReactPage.cs

```diff
@@ -167,6 +167,17 @@ protected virtual ReactRootView CreateRootView()
         }
 
         /// <summary>
+        /// Called during OnCreate, to set the RootView's background
+        /// </summary>
+        /// <remarks>
+        /// Subclasses may override this method if it needs to use a custom background.
+        /// </remarks>
+        protected virtual void SetRootViewBackground()
+        {
```

Comment:
can you make this a virtual setter that takes a Brush, and put the same virtual property in the WPF ReactPage as well?

It may also make sense to have the name be BackgroundBrush instead of exposing the internal detail that its setting a property on RootView.